sphinx/tutorial/rose/furthertopics/rose-stem.rst

```diff
@@ -202,12 +202,17 @@ class. An app may begin with a section like this:
    [ana:config]
    grepper-report-limit=5
    skip-if-all-files-missing=.true.
+   threads=10
 
 Each of these modifies the behaviour of ``grepper``. The first option
 suppresses printed output for each analysis task once the specified number
 of lines have been printed (in this case 5 lines). The second option
```

Comment:
of lines have been printed (in this case 5 lines). The  ``skip-if-all-files-missing`` option